src/validataclass/validators/allow_empty_string.py

```diff
@@ -0,0 +1,80 @@
+"""
+validataclass
+Copyright (c) 2021, binary butterfly GmbH and contributors
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE file.
+"""
+
+from copy import deepcopy
+from typing import Any, Optional
+
+from validataclass.exceptions import InvalidTypeError
+from .validator import Validator
+
+__all__ = [
+    'AllowEmptyString',
+]
+
+
+class AllowEmptyString(Validator):
+    """
+    Special validator that wraps another validator, but allows empty string ('') as input value.
+
+    By default, the wrapper returns empty string ('') for empty string ('') as input value. Optionally a default value
+    can be specified in the constructor that will be returned instead of empty string ('').
+
+    If the wrapped validator raises an `InvalidTypeError`, `AllowEmptyString` will add empty string ('')
+    to its 'expected_types' parameter and reraise it.
```

Comment:
See first comment: This would have to be updated to say "will add `str` ..." after that change.

Reply:
Changed